.github/workflows/deploy.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
       - uses: actions/setup-node@v3
         with:
-          node-version-file: ".node-version"
+          node-version-file: ".tool-versions"
```

Comment:
Nice, I like seeing the ADR affecting it's own PR as well since the new files are here as well.